lms/djangoapps/verify_student/tests/test_models.py

```diff
@@ -486,62 +485,86 @@ def test_verification_status_for_user(self, enrollment_mode, status, output):
 class VerificationCheckpointTest(ModuleStoreTestCase):
     """Tests for the VerificationCheckpoint model. """
 
-    MIDTERM = "midterm"
-    FINAL = "final"
-
     def setUp(self):
         super(VerificationCheckpointTest, self).setUp()
         self.user = UserFactory.create()
         self.course = CourseFactory.create()
+        self.checkpoint_midterm = u'i4x://{org}/{course}/edx-reverification-block/midterm_uuid'.format(
+            org=self.course.id.org, course=self.course.id.course
+        )
+        self.checkpoint_final = u'i4x://{org}/{course}/edx-reverification-block/final_uuid'.format(
+            org=self.course.id.org, course=self.course.id.course
+        )
 
-    @ddt.data(MIDTERM, FINAL)
-    def test_get_verification_checkpoint(self, check_point):
-        """testing class method of VerificationCheckpoint. create the object and then uses the class method to get the
-        verification check point.
+    @ddt.data("midterm", 'final')
```

Comment:
Nitpick: Try to avoid mixing quote levels like this; stick to single quotes, since they are what appear to be used throughout this module.